scripts/build_ckan_dataset_description.py

```diff
@@ -64,6 +64,8 @@ def build_ckan_dataset_description(strOutputFileName, number_of_objects, strFilt
         ckan_dataset_description["url"] = "https://ourairports.com/data"
     elif source == "healthsites":
         ckan_dataset_description["url"] = "https://www.healthsites.io/"
+    elif source == "unocha":
```

Comment:
it seems that value for ckan_dataset_description["url"] field can be added also.
it seems that `ckan_dataset_description["url"]=https://data.humdata.org/organization/un-ocha""`
will do